namespaces.go

```diff
@@ -59,3 +67,7 @@ func (c *Client) DeleteNamespace(appID, namespaceID string) error {
 	err := c.request("DELETE", "/apps/"+appID+"/namespaces/"+namespaceID, nil, nil)
 	return err
 }
+
+func NewBatchingInterval(interval int) *int {
```

Comment:
```suggestion
func BatchingInterval(interval int) *int {
```

Reply:
Fixed up in 89fdb1e9ecc10d398f9a2853a87aa9870fcc0d80